SnappyAPITests/SnappyAPITests.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  SnappyAPITests.swift
+//  SnappyAPITests
+//
+//  Created by Pawel Chmiel on 11.10.2016.
+//  Copyright © 2016 Droids On Roids. All rights reserved.
+//
+
+import XCTest
+import Alamofire
+@testable import Snappy
+
+class SnappyAPITests: XCTestCase {
+    
+    func testGetAllImages() {
+        let expectationGetAllImages = self.expectation(description: "getting all images")
+        
+        SnapchatAPI.getImages { (response) in
+            switch response {
+            case .success(let response):
+                let responseDict = response as? [String: Any]
+                let imageArray = responseDict?["images"] as? [[String: Any]]
+                XCTAssertTrue(imageArray!.count > 0)
+                expectationGetAllImages.fulfill()
```

Comment:
hmm, maybe move it after the switch? in failure case it doesn't get fulfilled and waits for response